### DISCUSSION TITLE
Proposal: Usage of `ledger-thorchain-js`

### DISCUSSION THREAD

**Discussion by veado** [shapeshift/hdwallet#495] [General]
[`ledger-thorchain-js`](https://github.com/thorchain/ledger-thorchain-js) will be the "official" JS Ledger library for THORChain.  Since `hdwallet` is already using [`@ledgerhq/hw-app-btc`](https://www.npmjs.com/package/@ledgerhq/hw-app-btc) and [`@ledgerhq/hw-app-eth`](https://www.npmjs.com/package/@ledgerhq/hw-app-eth), `ledger-thorchain-js` might be a good  option as well.

**Comment by mrnerdhair** [comment]
Thanks for opening this issue -- I know we talked in Discord but this definitely helps track things better. I expect we'll be looking to add support for the THORChain Ledger app soon, and this may be the best way.
